src/main/java/org/isf/medicalstockward/gui/WardPharmacyRectify.java

```diff
@@ -581,8 +581,6 @@ private MedicalWard chooseLot(Medical medical, boolean newLot) {
 					jLabelLotQty.setText(medWard.getQty().toString());
 					jLabelInLot.setVisible(true);
 					selectedLot = medWard.getLot();
-					selectedLot.setQuantity(medWard.getQty().intValue());
-					
 				}
 			} while (drugChooseList == null);
```

Comment:
Looking at the code the variable `drugChooseList` can never be null; see for example lines 536, 540, 562, and 576.  Thus the test of the `while` condition is always `false`.

Reply:
Removed